cmd/ncproxy/hcn.go

```diff
@@ -300,13 +301,29 @@ func hcnNetworkToNetworkResponse(network *hcn.HostComputeNetwork) (*ncproxygrpc.
 		DefaultGatewayIpv6:        defaultGatewayIPv6,
 	}
 
+	var startMac, endMac string
+	switch n := len(network.MacPool.Ranges); {
+	case n < 1:
+		return nil, fmt.Errorf("network %s(%s) MAC pool is empty", network.Name, network.Id)
+	case n > 1:
+		log.G(ctx).WithField("networkName", network.Name).Debug("network has multiple MAC pools, only returning the first")
```

Comment:
Should we have this be a warn instead of a debug?

Reply:
done